data/json/items/comestibles.json

```diff
@@ -7245,7 +7271,7 @@
     "spoils_in": 160,
     "charges": 5,
     "fun": -1,
-    "flags": [ "NO_FREEZE" ],
+    "flags": [ "FREEZERBURN" ],
```

Comment:
Mayo freezes around -50F. Though the water freezes separately earlier so maybe this is a good number.